src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageSession.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.Azure.ServiceBus.Amqp
     using System.Threading.Tasks;
     using Azure.Amqp;
     using Core;
-    using Messaging.Amqp;
 
     class AmqpMessageSession : MessageSession
```

Comment:
You're saying for the sake of thoroughness, to follow the same pattern with `MessageSession` and `AmqpMessageSession`? Where we remove `AmqpMessageSession` and add it's functionality to `MessageSession`? Just double checking...